app/controllers/conviction_rejection_forms_controller.rb

```diff
@@ -26,4 +29,8 @@ def authorize_action(transient_registration)
   def reject_renewal
     @transient_registration.conviction_sign_offs.first.reject!
   end
+
+  def success_path
```

Comment:
I don't think this is needed. Now we pass the value in as a hash, we have the key `success_path:` to tell us what the value relates to.